### PR TITLE
Minor Code cleanup for ExpressionVerifier

### PR DESCRIPTION
**Pull request by Praveen2112**
- Use `Iterables#getOnlyElement` for fetching single value
- Simplify `ExpressionVerifier#process` using `Stream#zip`

### CODE REVIEW

File: presto-main/src/test/java/io/prestosql/sql/planner/assertions/ExpressionVerifier.java [hunk 1]
```diff
@@ -415,12 +417,7 @@ protected Boolean visitInListExpression(InListExpression actual, Node expected)
         if (actuals.size() != expecteds.size()) {
             return false;
         }
-        for (int i = 0; i < actuals.size(); i++) {
-            if (!process(actuals.get(i), expecteds.get(i))) {
-                return false;
```
Comment (kokosing) [inline presto-main/src/test/java/io/prestosql/sql/planner/assertions/ExpressionVerifier.java]: this loop is a bit easier to debug as one can set a breakpoint here to see why it return false. However it is also easy to do it with streams.

File: presto-main/src/test/java/io/prestosql/sql/planner/assertions/ExpressionVerifier.java [hunk 2]
```diff
@@ -415,12 +417,7 @@ protected Boolean visitInListExpression(InListExpression actual, Node expected)
...
-            }
-        }
-        return true;
+        return zip(actuals.stream(), expecteds.stream(), this::process).allMatch(x -> x);
```
Comment (kokosing) [inline presto-main/src/test/java/io/prestosql/sql/planner/assertions/ExpressionVerifier.java]: nits:
 - move `.allMatch` to new line. 
 - use Function.identity()

File: presto-main/src/main/java/io/prestosql/util/MorePredicates.java [hunk 3]
```diff
@@ -29,4 +29,9 @@ private MorePredicates() {}
         }
         return predicate;
     }
+
+    public static Predicate<Boolean> identity()
```
Comment (findepi) [inline presto-main/src/main/java/io/prestosql/util/MorePredicates.java]: This cannot be static imported together with `java.util.function.Function#identity`.

File: presto-main/src/test/java/io/prestosql/sql/planner/assertions/ExpressionVerifier.java (hunk 1 above)
Comment (findepi) [reply presto-main/src/test/java/io/prestosql/sql/planner/assertions/ExpressionVerifier.java]: No, IMO it's not nearly as easy to debug with streams, not to mention "step into".

File: presto-main/src/main/java/io/prestosql/util/MorePredicates.java (hunk 3 above)
Comment (Praveen2112) [reply presto-main/src/main/java/io/prestosql/util/MorePredicates.java]: can we rename the function to `identityPredicate` ?

**Pull request closed without merging**